internal/grid/grid.go

```diff
@@ -38,7 +38,7 @@ const (
 	minBufferSize = 1 << 10
 
 	// defaultBufferSize is the default buffer allocation size.
-	defaultBufferSize = 4 << 10
+	defaultBufferSize = 8 << 10
```

Comment:
Most messages are significantly below 4K. this is just wasting memory.

```suggestion
	defaultBufferSize = 4 << 10
```